babel.config.json

```diff
@@ -1,4 +0,0 @@
-{
-  "plugins": ["@babel/plugin-transform-runtime"],
-  "presets": [["@babel/preset-env"]]
-}
```

Comment:
I think we should keep babel to build the project. It needs to use [preset/typescript](https://babeljs.io/docs/en/babel-preset-typescript)